Makefile

```diff
@@ -0,0 +1,55 @@
+dpl ?= deploy.env
+include $(dpl)
+ENVIRONMENT=$(shell sed 's/=.*//' $(dpl));
+export ENVIRONMENT
+
+BUILD_NUMBER ?= local-build
+GIT_COMMIT ?= local-commit
+BRANCH_NAME ?= local-branch
+
+VERSION=$(shell docker-compose run --rm smartcall version)
```

Comment:
I think this will fail.

In this project's case, we can stick it in deploy.env and it will get spat into the environment here.